src/main/resources/fxml/addvault_new_expert_settings.fxml

```diff
@@ -0,0 +1,81 @@
+<?import javafx.scene.control.Hyperlink?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.Tooltip?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.Region?>
+<?import javafx.scene.layout.StackPane?>
+<?import javafx.scene.layout.VBox?>
+<VBox xmlns:fx="http://javafx.com/fxml"
+	  xmlns="http://javafx.com/javafx"
+	  fx:controller="org.cryptomator.ui.addvaultwizard.CreateNewVaultExpertSettingsController"
+	  prefWidth="450"
+	  prefHeight="450"
+	  spacing="12"
+	  alignment="CENTER_LEFT">
+	<padding>
+		<Insets topRightBottomLeft="24"/>
+	</padding>
+	<children>
+		<Region prefHeight="12" VBox.vgrow="NEVER"/>
+		<Label text="%addvaultwizard.new.expertSettings.summary.introduction" wrapText="true"/>
+		<Label fx:id="vaultNameLabel" alignment="CENTER_RIGHT" graphicTextGap="6" wrapText="true">
+			<graphic>
+				<FontAwesome5IconView styleClass="glyph-icon-muted" glyph="PENCIL"/>
+			</graphic>
+		</Label>
+		<Label fx:id="vaultPathLabel" alignment="CENTER_RIGHT" graphicTextGap="6" wrapText="true">
+			<graphic>
+				<FontAwesome5IconView styleClass="glyph-icon-muted" glyph="HDD"/>
+			</graphic>
+		</Label>
+		<Region prefHeight="12" VBox.vgrow="NEVER"/>
+		<CheckBox fx:id="expertSettingsCheckBox" text="%addvaultwizard.new.expertSettings.enableExpertSettingsCheckbox" onAction="#toggleUseExpertSettings"/>
+		<VBox spacing="6" visible="${expertSettingsCheckBox.selected}">
+			<HBox spacing="2" HBox.hgrow="NEVER">
+				<Label text="%addvaultwizard.new.expertSettings.shorteningThreshold.title"/>
+				<Region prefWidth="2"/>
+				<Hyperlink contentDisplay="GRAPHIC_ONLY" onAction="#openDocs">
+					<graphic>
+						<FontAwesome5IconView glyph="QUESTION_CIRCLE" styleClass="glyph-icon-muted"/>
+					</graphic>
+					<tooltip>
+						<Tooltip text="%addvaultwizard.new.expertSettings.shorteningThreshold.tooltip" showDelay="10ms"/>
+					</tooltip>
+				</Hyperlink>
+			</HBox>
+			<HBox>
+				<NumericTextField fx:id="shorteningThresholdTextField"/>
+				<Region prefWidth="4" HBox.hgrow="NEVER"/>
+				<StackPane>
+					<Label styleClass="label-muted" text="%addvaultwizard.new.expertSettings.shorteningThreshold.invalid" textAlignment="RIGHT" alignment="CENTER_RIGHT" visible="${!controller.validShorteningThreshold}" graphicTextGap="6">
+						<graphic>
+							<FontAwesome5IconView styleClass="glyph-icon-red" glyph="TIMES"/>
+						</graphic>
+					</Label>
+					<Label styleClass="label-muted" text="%addvaultwizard.new.expertSettings.shorteningThreshold.valid" textAlignment="RIGHT" alignment="CENTER_RIGHT" visible="${controller.validShorteningThreshold}" graphicTextGap="6">
+						<graphic>
+							<FontAwesome5IconView styleClass="glyph-icon-primary" glyph="CHECK"/>
+						</graphic>
+					</Label>
+				</StackPane>
+			</HBox>
+			<Label styleClass="label-muted" text="%addvaultwizard.new.expertSettings.shorteningThreshold.message" visible="${!controller.validShorteningThreshold}"/>
```

Comment:
in the AddVaultLocation scene the valid indicator is below the textfield and if the path is invalid, the reason is written next to the icon (not an extra message).

@mindmonk What were your thoughts of choosing this layout?

Reply:
I didn't want to use the full window width for a text field that can contain a maximum of 3 characters, so I chose a short text field and placed the 'valid' message next to it. The additional information was initially continuously visible and later tied to the validity. Should I use the full width or is the following solution sufficient?
<img width="730" alt="Bildschirmfoto 2023-07-11 um 14 35 17" src="https://github.com/cryptomator/cryptomator/assets/29817198/aa6d72c7-7be7-43d1-9b5b-1abd25ce4e32">
<img width="730" alt="Bildschirmfoto 2023-07-11 um 14 35 23" src="https://github.com/cryptomator/cryptomator/assets/29817198/29774983-4c8c-4119-84a1-1870364e3f56">

Reply:
Oh, you were too fast with the change^^ Either revert https://github.com/cryptomator/cryptomator/pull/2987/commits/5f62f038d502288c62c9b5c0ff442dccf7757066 or change the Label such, that it is always displayed below the textfield.

The reason is localization and a wiggly size of the textinput field. Regarding the first, other languages might use even more space, making the label looking really wierd.